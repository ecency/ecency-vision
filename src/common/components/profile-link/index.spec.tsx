import React from "react";

import ProfileLink from "./index";
import TestRenderer from "react-test-renderer";
import { createBrowserHistory } from "history";

it("(1) Render", () => {
  const props = {
    history: createBrowserHistory(),
    children: <span>username</span>,
    username: "username",
    addAccount: () => {},
  };
  const renderer = TestRenderer.create(<ProfileLink {...props} />);
  expect(renderer.toJSON()).toMatchSnapshot();
});
