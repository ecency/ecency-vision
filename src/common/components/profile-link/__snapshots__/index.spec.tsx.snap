// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(1) Render 1`] = `
<a
  href="/@username"
  onClick={[Function]}
>
  <span>
    username
  </span>
</a>
`;
